lib/rspec/rails/adapters.rb

```diff
@@ -136,6 +136,24 @@ def method_name
       end
     end
 
+    # @private
+    #
+    # Reset ActiveSupport::CurrentAttributes only *after* tests run, rather
```

Comment:
True. That is a shortcoming of the approach used in 6.1.3 already, of course, since it only includes the ActiveSupport helper inside RailsExampleGroup.

Do you have a "feel" for where else in the hierarchy we might _include this module_ to achieve a wider area of effect? It can be put just about anywhere, especially with some `defined?` and/or `respond_to?` guards, and should be pretty bomb-proof. Once we've figured that out, we can decide if the adapter module's chosen namespace, as a Rails adapter, is correct or if it should also move elsewhere in the code base.